src/Okta.Idx.Sdk.E2ETests/Helpers/A18NClient/A18nClient.cs

```diff
@@ -305,17 +305,17 @@ private string GetRelativeUri(string resource = default)
             {
                 return $"{RelativePart}/{resource}";
             }
-        }
-
-        private void CleanUpOldUsedProfiles(string newProfileUniqueTag)
-        {
-            var activeProfiles = GetActiveProfilesAsync().Result;
-
-            foreach (var profile in activeProfiles.Profiles.Where(p => p.DisplayName == newProfileUniqueTag))
-            {
-                DeleteProfileAsync(profile.ProfileId).Wait();
-            }
-        }
+        }
+
+        private void CleanUpOldUsedProfiles(string newProfileUniqueTag)
```

Comment:
Should this be called `CleanUpOldUnUsedProfiles`, or maybe `CleanUpOldProfiles`, or even `CleanUpProfiles`?

Actually, looking at what the code is doing maybe this could be called `DeleteDuplicates`.

Reply:
@bryanapellanes-okta I use `DisplayName` property of A18N profile to tag the profiles our test creates, this lets us to find and delete abandoned profiles which were created during previous test run, and not deleted because of some kind of exception during the test.

The purpose of this function is to make sure there are no leftovers from the last test. 
I don't think `DeleteDuplicates` is the right name for it, but I'm not sure about others. Do you think `CleanUpProfiles` is better?

Reply:
I changed the name to `CleanUpOldProfiles`